src/stories/Library/Lists/list-materials/list-materials.scss

```diff
@@ -1,5 +1,7 @@
 .list-materials {
   @include show-svg-on-hover(".list-dashboard__arrow");
+  @include show-svg-on-hover(".list-materials__arrow");
+  @include show-svg-on-hover(".arrow-button");
 
```

Comment:
Without having dived into the definitions of `show-svg-on-hover(".list-materials__arrow")` and `show-svg-on-hover(".arrow-button")`: Does it make sense to have both. Couldn't it be handled with only one definition?

Reply:
Short answer: no.
Long answer: The DOM structure can vary between the different types of cards we use, and the container in which the arrow component is located can have any of these classes. So we need to make sure we target them with `show-svg-on-hover()`